src/modules/pages/components/ExpenditurePage/ExpenditurePage.tsx

```diff
@@ -26,6 +26,8 @@ import LockedPayments from '~dashboard/ExpenditurePage/Payments/LockedPayments';
 import { useLoggedInUser } from '~data/helpers';
 import { Recipient } from '~dashboard/ExpenditurePage/Payments/types';
 import LockedExpenditureSettings from '~dashboard/ExpenditurePage/ExpenditureSettings/LockedExpenditureSettings';
+import { useDialog } from '~core/Dialog';
+import EscrowFundsDialog from '~dashboard/Dialogs/EscrowFundsDialog/EscrowFundsDialog';
```

Comment:
You need to add index.ts file to the `EscrowFundsDialog` folder & to export `EscrowFundsDialog` file from there. Then remove the last `EscrowFundsDialog` from this line